src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    chipVersion: version,
+    wheels: wheels,
```

Comment:
if `variableName === keyName` you can use short syntax:
```suggestion
    name,
    chipVersion: version,
    wheels,
```

Reply:
ready